test-service/rebar.config

```diff
@@ -1,14 +1,14 @@
 {erl_opts, [debug_info]}.
 {deps, [
-    {shotgun, "0.5.0"},
```

Comment:
Needed to use 1.0.1 to support TLS as implementeed in the SDK. This should have been updated earlier.